packages/integration-tests/projects/suite-web/tests/dashboard/assets.test.ts

```diff
@@ -0,0 +1,29 @@
+// @group:suite
+// @retry=2
+// 1. navigate to the `Dashboard` page
+// 2. check the `Assets` part of the page
+// 1. the modal is rendered correctly and shows 1-n coins
+// 3. click on a coin name (eg `Bitcoin`)
+// 1. user is transferred to a bitcoin account
+
+describe('Dashboard', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic: 'all all all all all all all all all all all all',
+        });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/');
+        cy.passThroughInitialRun();
+    });
+
+    it('Assets', () => {
+        // waiting until discovery finishes
+        cy.discoveryShouldFinish();
+        // opens btc account through assets table
+        cy.contains('div[class*="AssetTable"]', 'Bitcoin').should('be.visible').click();
```

Comment:
please try to avoid this kind of selectors as much as possible. use `data-test` custom attrs and dont be afraid to ask us to add them if you are unsure how to do it.

Reply:
In this situation, I would even suggest removing the first pameter of the `.contains()` command entirely.

Reply:
ok, I see, thank you!